src/containers/MainScreen/index.js

```diff
@@ -1,25 +1,29 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import { View, Text, Button } from 'react-native';
-import { connect } from 'react-redux';
-import { object, func } from 'prop-types';
+import { useDispatch } from 'react-redux';
 
 import { logout } from 'actions/userActions';
 import translate from 'utils/i18n';
+import useSession from 'hooks/useSession';
 import styles from './styles';
 
-const MainScreen = ({ user: { email }, logout }) => (
-  <View style={styles.container}>
-    <Text>Hey {email}, you&#39;re logged in!</Text>
-    <Button
-      onPress={logout}
-      title={translate('MAIN_SCREEN.logout')}
-    />
-  </View>
-);
+const MainScreen = () => {
+  const dispatch = useDispatch();
```

Comment:
A small detail. Can you add a line as you do in the signup screen?

Reply:
fixed!